services/gitea/9.3.0/defaults/cm.yaml

```diff
@@ -58,6 +58,10 @@ data:
         periodSeconds: 10
         successThreshold: 1
         failureThreshold: 10
+      # We use startupProbe unlike the default configuration,
+      # to shorten readiness time, set initialDelaySeconds to 5 seconds.
+      livenessProbe:
+        initialDelaySeconds: 5
```

Comment:
how long is the default value (curiosity)?

Reply:
gitea.livenessProbe.initialDelaySeconds | Initial delay before liveness probe is initiated | 200